fuzzing/utils/fuzz_method_utils.go

```diff
@@ -32,3 +35,24 @@ func IsPropertyTest(method abi.Method, prefixes []string) bool {
 	}
 	return false
 }
+
+// IsAssertionTest checks whether the method is configured by the attached fuzzer to be a target of assertion testing.
+// Returns true if this target should be tested, false otherwise.
+func isAssertionTest(method abi.Method, includeViewMethods bool) bool {
+	// Only test constant methods (pure/view) if we are configured to.
+	return !method.IsConstant() || includeViewMethods
+}
```

Comment:
i am going to get rid of this function because technically, from an API-level, an assertion test is an assertion test iff (1) it's not a property test, (2) it's not an optimization test, (3) it's state changing OR you are testing view methods.

So technically this function does not satisfy (1) or (2). I will push up the changes soon.